oshdb/src/main/java/org/heigit/ohsome/oshdb/util/bytearray/ByteArrayOutputWrapper.java

```diff
@@ -63,12 +63,13 @@ public void writeU64(long value) {
    * @param value current value
    * @param last last value or delta encoding
    */
-  public void writeU64Delta(long value, long last) {
+  public long writeU64Delta(long value, long last) {
```

Comment:
maybe add a `@returns` section to the javadoc to document that the returned value is the same as the input `value`?!